lua/starfall/sflib.lua

```diff
@@ -892,6 +892,22 @@ function SF.CheckMaterial(material)
 	return mat
 end
 
+
+function SF.CheckModel(model, player)
+	if #model > 260 then return false end
+	model = SF.NormalizePath(string.lower(model))
+	if util.IsValidModel(model) and util.IsValidProp(model) then
+		if player and player:IsValid() then
+			if hook.Run("PlayerSpawnObject", player, model)~=false then
```

Comment:
💡 While this is up to you since it's your codebase, I find early return/"happy path" code much easier to read since it (usually) removes all the nested if statements
eg
```lua
if badthing then
    return false
elseif otherbadthing then
    return false
end
return true
```

Reply:
Mine's the same, just opposite.

```lua
if goodthing then
    return true
elseif othergoodthing then
    return true
end
return false
```